test/ui/DateUtilsTest.ts

```diff
@@ -0,0 +1,59 @@
+import { DateUtils } from '../../src/utils/DateUtils';
+import { IDateToStringOptions } from '../../src/utils/DateUtils';
+import * as moment from 'moment';
+import { l } from '../../src/strings/Strings';
+
+export function DateUtilsTest() {
+
+  describe('DateUtils', () => {
+    let options: IDateToStringOptions;
+    afterEach(() => {
+      options = null;
+      String['locale'] = 'en';
+    });
+
+    it('should use the correct locale if the locale is "no"', () => {
+      String['locale'] = 'no';
+      DateUtils.setLocale();
+      expect(DateUtils.dateToString(DateUtils.convertToStandardDate('1980-02-11'), { now: moment('1980-02-11').toDate() })).toEqual(l('Today'));
```

Comment:
When you see that your UT are repeating a lot of stuff to "setup" correctly, you should spot these pattern and instead execute them in the `beforeEach` function of the test suite.

So for example, you can setup the `now` variable only once in the `beforeEach` of this suite.